warehouse/static/sass/blocks/_notification-bar.scss

```diff
@@ -54,6 +58,25 @@
     }
   }
 
+  &__dismiss {
+    position: absolute;
+    top: 0;
+    right: 0;
+    cursor: pointer;
+    font-size: 1.5rem;
+  }
+
+  // Indicates that a notification is dismissable.
+  // Since this defaults to not displaying the element, one must make sure the
+  // notification controller is being used.
+  &--dismissable {
+    display: none;
+  }
```

Comment:
it might be nice to add a CSS transition here, so it fades in :)

Reply:
I'm not too familiar with animations -- which property should we be animating? Based on [this Stack Overflow question](https://stackoverflow.com/a/3332179/6553469), we can't transition the `display` property. Should we have it transition `opacity` and/or `height` instead?

Reply:
yes, you're right.  Let's leave it for now :)